certbot/plugins/standalone_test.py

```diff
@@ -2,12 +2,16 @@
 import argparse
 import socket
 import unittest
+from socket import errno as socket_errors  # type: ignore
```

Comment:
Is there an issue tracking the need to use `type: ignore` here? Can we link it?

Reply:
# https://github.com/python/typeshed/blob/master/stdlib/2and3/socket.pyi

Reply:
https://github.com/python/typeshed/blob/master/stdlib/2and3/socket.pyi